.circleci/release.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
```

Comment:
Let's only use the `.circleci` directory for CircleCI related files. I think we should move this to a `Scripts` directory at the root level.